src/commands/run.js

```diff
@@ -43,12 +45,27 @@ export async function run({ taskName, options }) {
   if (logger instanceof InteractiveLogger) logger.clearTasks()
 
   const failedTasks = tasks.allFailedTasks()
-
   if (failedTasks.length > 0) {
     logger.fail(`Failed tasks: ${failedTasks.join(', ')}`)
   }
 
-  if (Object.entries(tasks.allTasks).every(([, task]) => task.status === 'success:lazy')) {
-    logger.log('\n' + rainbow('>>> MAXIMUM LAZY'))
+  const stats = tasks.getTaskStats()
+  const successOutput = `${kleur.green(stats.successful.toString() + ' successful')}, ${
+    stats.allTasks
+  } total`
+
+  let output = ''
+  if (stats['success:lazy'] === stats.allTasks) {
+    output = dedent`
+    
+          Tasks:     ${successOutput}
+         Cached:     ${rainbow('>>> MAXIMUM LAZY')}`
```

Comment:
Still uses `Cached:` prefix. Felt I could not get the alignment right without it, but we can drop it if you'd like.